docs/sites.yml

```
@@ -4516,3 +4516,14 @@
   built_by: Nick Taylor
   built_by_url: https://www.iamdeveloper.com
   featured: false
+- title: Keziah MOSELLE blog
```

Would it be OK if we normalize the case here?

```suggestion
- title: Keziah Moselle Blog
```